src/StockportWebapp/Views/stockportgov/Shared/DirectorySearchResult.cshtml

```diff
@@ -31,7 +31,9 @@
                     }
 
                     <h3 class="navcard__title navcard__title--large">@if (Model.ShowMapPin && Model.IsPinned){<partial name="MapPin" view-data='new ViewDataDictionary(ViewData) { { "Count", Model.MapPinIndex}, {"Highlighted", true } }' />}
-                        else if (Model.ShowMapPin){<partial name="MapPin" view-data='new ViewDataDictionary(ViewData) { { "Count", Model.MapPinIndex}, {"Highlighted", false } }' />}@Model.DirectoryEntry.Name
+                        else if (Model.ShowMapPin){<partial name="MapPin" view-data='new ViewDataDictionary(ViewData) { { "Count", Model.MapPinIndex}, {"Highlighted", false } }' />}
+                        @*else
+                        {<partial name="MapPinBlank" />}*@@Model.DirectoryEntry.Name
```

Comment:
does this commented out bit need to be here

Reply:
Removed